packages/flame/CHANGELOG.md

```diff
@@ -31,6 +31,12 @@
  - Add class `NotifyingVector2`
  - Add class `Transform2D`
  - Remove `FPSCounter` from `BaseGame`
+ - Refactor `PositionComponent` to work with `Transform2D`. Several methods were updated to work
```

Comment:
I think it would be cleaner to split up the other sentence into multiple entries.

- renderFlipX replace by..
- angleTo removed...
- positioncomponent debug...

Reply:
sure, will do